loadbalancer/algorithm.go

```diff
@@ -254,13 +279,17 @@ func computeLoadAverage(ctx *routing.LBContext) float64 {
 }
 
 // Returns index of endpoint with closest hash to key's hash, which is also below the target load
-func (ch consistentHash) boundedLoadSearch(key string, balanceFactor float64, ctx *routing.LBContext) int {
-	ringIndex := ch.searchRing(key)
+// endpointIsEligible function is used to skip endpoints we don't want, such as fading endpoints
```

Comment:
maybe rename `endpointIsEligible` to `skipEndpoint`

Reply:
Grand that would be clearer :+1: